### PR TITLE
Replaced all the css into tailwind css on FlatCard  component

### PR DESCRIPTION
**Pull request by subhajit20**
# Description of changes

Replaced all the css into tailwind css on FlatCard  component
Issue - #1725 

Before:
Before

After:
After

# Issue Resolved

Fixes #NA

### CODE REVIEW

**Review by kylemh — COMMENTED** [review]
it's easier for me to review if you delete the CSS file so I can compare in the diff view!

**Comment by subhajit20** [comment]
> it's easier for me to review if you delete the CSS file so I can compare in the diff view!

hey If I delete the CSS file then one class will be corrupted `{styles.children}` then what to do ?

**Comment by kylemh** [comment]
There doesn't appear to be any `.children` class in `FlatCard.module.css`, so I think you're safe to delete that reference so:

`<div className={styles.children}>` can become `<div>`

**Comment by subhajit20** [comment]
> There doesn't appear to be any `.children` class in `FlatCard.module.css`, so I think you're safe to delete that reference so:
> 
> `<div className={styles.children}>` can become `<div>`

Yes I have deleted the css file.

File: components/Cards/FlatCard/FlatCard.js [hunk 1]
```diff
@@ -28,16 +27,19 @@ function FlatCard({ button: Button, children, className, header, image }) {
...
+      <div className="p-8 border-4 border-solid border-themePrimary">
+        {header && <div className="order-2 text-center">{header}</div>}
 
         {hasImage && (
-          <div className={styles.rowCenter}>
-            <div data-testid={FLAT_CARD_IMAGE} className={styles.imageWrapper}>
+          <div className="flex justify-center">
+            <div
+              data-testid={FLAT_CARD_IMAGE}
+              className="border-solid border-themePrimary order-1 absolute -top-1 h-52 [&>img]:object-cover"
```
Comment (kylemh) [inline components/Cards/FlatCard/FlatCard.js:41]:
```suggestion
              className="border-solid border-themePrimary border-[3px] order-1 absolute -top-4 h-52 [&>img]:object-cover"
```

File: components/Cards/FlatCard/FlatCard.js [hunk 2]
```diff
@@ -50,11 +52,15 @@ function FlatCard({ button: Button, children, className, header, image }) {
           </div>
         )}
 
-        {header && <hr className={styles.divider} />}
+        {header && <hr className="w-10/12 border-1 border-solid border-themePrimary" />}
```
Comment (kylemh) [inline components/Cards/FlatCard/FlatCard.js:55]:
```suggestion
        {header && <hr className="w-10/12 border-solid border-themePrimary" />}
```

**Review by kylemh — APPROVED** [review]

**Pull request merged**